11/umbraco-cms/extending/packages/types-of-packages.md

```diff
@@ -1,19 +1,62 @@
+---
+description: Depending on which version of Umbraco CMS you are using, we distinguish between two package types, NuGet packages, and Package ZIP files.
+---
+
 # Package Types
 
-There are two main ways to install packages in Umbraco: Package zip files and NuGet packages. Each of these has its own advantages.
+There are two main ways to install packages in Umbraco CMS: Package zip files and NuGet packages. Each of these has its own advantages.
```

Comment:
```suggestion
There are two ways to install packages in Umbraco CMS: Package zip files and NuGet packages.
```

I'm thinking we should remove the "advantages and disadvantages" here, as it's a bit moot given all modern Umbraco packages can only be NuGet packages.